tests/datalab/test_datalab.py

```diff
@@ -676,7 +689,7 @@ def test_report_for_outlier_issues_via_pred_probs(find_issues_kwargs):
     find_issues_kwargs["issue_types"] = {"outlier": {"k": 1}}
     lab.find_issues(**find_issues_kwargs)
 
-    reporter = Reporter(lab.data_issues, verbosity=0, include_description=False)
+    reporter = Reporter(lab.data_issues, imagelab=None, verbosity=0, include_description=False)
```

Comment:
If we decouple `Reporter` from `Imagelab`, then maybe this argument won't be necessary.